common/source/docs/common-spracingh7-extreme.rst

```diff
@@ -0,0 +1,155 @@
+     :target: ../_images/SPracing-H7-extreme-pinout.png
+
+.. note:: be sure to configure the voltage selection pads for the camera, vtx
+
+VTX Supply Switch
+=================
+Set the ``RELAYx_PIN`` to "81" for on/off of VTX power pad.
+
+Then select an RC channel for control (Chx) and set its ``RCx_OPTION`` to the appropriate Relay (1-4) that you had set its pin parameter above.
+
+For example, use Channel 10 to control the power using Relay 2:
+
+- :ref:`RELAY_PIN2<RELAY_PIN2>` = "81" (Relay 2 is set by default, but can be changed)
+- :ref:`RC10_OPTION<RC10_OPTION>` = "34" (Relay2 Control)
+
+.. note:: setting the above Relay on/high will turn off that supply. If nothing is assigned, default is powered on.
+
+Default UART order
+==================
+
+- SERIAL0 = console = USB
+- SERIAL1 = Telemetry1 = USART1 (has DMA) normally RC input
+- SERIAL2 = Telemetry2 = USART2 (has DMA) 
+- SERIAL3 = GPS1 = USART3 (has DMA)
+- SERIAL4 = GPS2 = UART4 (has DMA)
+- SERIAL5 = USER = UART5 (has DMA)
+- SERIAL6 = USER = USART6 (available instead of PWM outputs 7(TX6) and 8(RX6) if :ref:`BRD_ALT_CONFIG<BRD_ALT_CONFIG>` = 1)
+- SERIAL8 = USER = UART8 
+
+Serial port protocols (Telem, GPS, etc.) can be adjusted to personal preferences.
+
+RC Input
+========
+RC input is configured on the R1 (UART1_RX) pin. It supports all RC protocols except PPM. See :ref:`common-rc-systems` for details for a specific RC system. :ref:`SERIAL1_PROTOCOL<SERIAL1_PROTOCOL>` is set to "23", by default, to enable this.
+
+- SBUS/DSM/SRXL connects to the R1 pin but SBUS requires that the :ref:`SERIAL1_OPTIONS<SERIAL1_OPTIONS>` be set to "3".
+
+- FPort requires connection to T6 and :ref:`SERIAL1_OPTIONS<SERIAL1_OPTIONS>` be set to "7".
+
+- CRSF also requires a T1 connection, in addition to R1, and automatically provides telemetry. Set :ref:`SERIAL1_OPTIONS<SERIAL1_OPTIONS>` to "0".
+
+- SRXL2 requires a connection to T1 and automatically provides telemetry.  Set :ref:`SERIAL1_OPTIONS<SERIAL1_OPTIONS>` to "4".
+
+Any UART can be used for RC system connections in ArduPilot also, and is compatible with all protocols except PPM. See :ref:`common-rc-systems` for details.
+
+OSD Support
+===========
+
+The SPracing H7 Extreme onboard OSD is used by setting :ref:`OSD_TYPE<OSD_TYPE>` 1 (MAX7456 driver).
+
+SmartPort Telemetry
+===================
+
+SmartPort (Sport) telemetry setup by default to be connected directly to UART2 TX pin with no external inverters required.
+ 
+Dshot capability
+================
+
+All motor/servo outputs are Dshot(except outputs 9/10/11) and PWM capable. However, mixing Dshot and normal PWM operation for outputs is restricted into groups, ie. enabling Dshot for an output in a group requires that ALL outputs in that group be configured and used as Dshot, rather than PWM outputs. The output groups that must be the same (PWM rate or Dshot, when configured as a normal servo/motor output) are: 1-4, 5/6, 7/8, 9/10.
```

Comment:
PWM 1-4 support bi-directional dshot

Reply:
fixed...and all outputs are DShot capable I think...another cut and paste error from KakuteH7